routes/duration.js

```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
+const express = require('express'); 
+const router = express.Router();
+const durationCtrl = require('../controllers/duration');
+
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
router.get("/", durationCtrl.index); 
```